spec/Errors.tex

```diff
@@ -130,6 +130,8 @@ \subsubsection{Catch clauses}
 \begin{chapelexample}{catching-errors.chpl}
 \begin{chapel}
 \begin{verbatim}
+use SysError;
```

Comment:
Let's not modify the .tex spec since PR #14303 will remove it and it's not currently used